lib/image-stream/index.js

```diff
@@ -108,6 +108,9 @@ exports.getFromFilePath = (file) => {
  */
 exports.getImageMetadata = (file) => {
   return exports.getFromFilePath(file).then((image) => {
+    // Since we're not consuming this stream,
+    // destroy() it, to avoid dangling open file descriptors etc.
+    image.stream.destroy();
```

Comment:
Can we also add a blank line below this? Otherwise I get the impression that the comment applies to this line, and to the one below.